public/components/TagEdit/formComponents/TagName.react.js

```diff
@@ -118,6 +118,16 @@ export default class TagNameEdit extends React.Component {
       return 'tracking/' + trackingTypeName + '/';
     }
 
+    // Paid content with sub type of hosted exception
+
+    if (this.props.tag.type === tagTypes.paidContent.name
+      && this.props.tag.paidContentInformation
+      && this.props.tag.paidContentInformation.paidContentType
```

Comment:
This line technically isn't required as if `.paidContentInformation` exists then `.paidContentType` would be `undefined` if it doesn't exist